### PR TITLE
Fix all 'context.exception.message' test verifications

### LINKED ISSUE #3644: Fix all 'context.exception.message' test verifications
Fix all blocks like:
```
with self.assertRaises(UIError) as context:
    action
    self.assertEqual(
        context.exception.message,
        'Exception text'
)
```
as `self.assertRaises` works in another way according to the documentation
```
with self.assertRaises(SomeException) as cm:
  do_something()
the_exception = cm.exception
self.assertEqual(the_exception.error_code, 3)
```
so we need to change our code to:
```
with self.assertRaises(UIError) as context:
    action
self.assertEqual(
    context.exception.message,
    'Exception text'
)
```

### PR DESCRIPTION
**Pull request by svtkachenko**
Fixes #3644

Test results:
```python
py.test -v -k "(ContentViewTestCase and (test_negative_add_components_to_non_composite or test_negative_add_dupe_repos)) or (SettingTestCase and test_negative_update_oauth_active_param) or (TemplateTestCase and (test_negative_create_without_type or test_negative_create_without_upload))" tests/foreman/ui
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.12, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /home/qui/code/robottelo/venv2/bin/python2.7
cachedir: .cache
rootdir: /home/qui/code/robottelo, inifile: 
plugins: cov-2.3.1, xdist-1.14
collected 969 items 

tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_components_to_non_composite <- robottelo/decorators/__init__.py PASSED
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_dupe_repos <- robottelo/decorators/__init__.py PASSED
tests/foreman/ui/test_setting.py::SettingTestCase::test_negative_update_oauth_active_param PASSED
tests/foreman/ui/test_template.py::TemplateTestCase::test_negative_create_without_type <- robottelo/decorators/__init__.py PASSED
tests/foreman/ui/test_template.py::TemplateTestCase::test_negative_create_without_upload <- robottelo/decorators/__init__.py PASSED

 964 tests deselected by '-k(ContentViewTestCase and (test_negative_add_components_to_non_composite or test_negative_add_dupe_repos)) or (SettingTestCase and test_negative_update_oauth_active_param) or (TemplateTestCase and (test_negative_create_without_type or test_negative_create_without_upload))' 
========================================================= 5 passed, 964 deselected in 231.37 seconds ==========================================================
```

### CODE REVIEW

**Comment by sghai** [comment]
ACK

File: tests/foreman/ui/test_template.py [hunk 1]
```diff
@@ -155,10 +155,10 @@ def test_negative_create_without_upload(self):
                     custom_really=True,
                     template_type='PXELinux template',
                 )
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not create blank template "{0}"'.format(name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not create blank template "{0}"'.format(name)
```
Comment (elyezer) [inline tests/foreman/ui/test_template.py]: prefix the formatting string with `u` name may contain unicode characters and that will fail on Python 2.

File: tests/foreman/ui/test_template.py [hunk 2]
```diff
@@ -131,10 +131,10 @@ def test_negative_create_without_type(self):
                     custom_really=True,
                     template_type='',
                 )
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not create template "{0}" without type'.format(name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not create template "{0}" without type'.format(name)
```
Comment (elyezer) [inline tests/foreman/ui/test_template.py]: prefix the formatting string with u name may contain unicode characters and that will fail on Python 2.

File: tests/foreman/ui/test_contentview.py [hunk 3]
```diff
@@ -716,11 +716,11 @@ def test_negative_add_dupe_repos(self):
...
             with self.assertRaises(UIError) as context:
                 self.content_views.add_remove_repos(cv_name, [repo_name])
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not find repo "{0}" to add into CV'
-                    .format(repo_name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not find repo "{0}" to add into CV'
```
Comment (elyezer) [inline tests/foreman/ui/test_contentview.py]: prefix the formatting string with u name may contain unicode characters and that will fail on Python 2.

**Comment by oshtaier** [comment]
ACK

**Pull request merged**